docs-md/concepts/events.md

```diff
@@ -1,15 +1,53 @@
 ---
 layout: page
+title: Events
+tagline: Events in ksqlDB
+description: Learn about events and stream processing in ksqlDB. 
+keywords: ksqldb, event, stream
 ---
 
-ksqlDB is an event streaming database purpose-built for stream processing applications. The main focus of stream processing is modeling computation over unbounded streams of events. But what is an event anyway?
+ksqlDB is an event streaming database that's purpose-built for stream
+processing applications. The main focus of stream processing is modeling
+computation over unbounded streams of events.
 
-An event is anything that happened or occurred. It could be something high-level that happened in a business, like the sale of an item or the submission of an invoice. Or it can be something like low-level, like a log line emitted by a web server when a request is received. Anything that happens at a point in time is an event.
+What is an event?
+-----------------
 
-Because events are so fundamental to stream processing, they are ksqlDB’s core unit of data. All of ksqlDB’s features are oriented around making it easy to solve problems using events. But while it’s easy to think about individual events, figuring out how to group related events together is a bit more challenging. Fortunately, the idea of storing related events is well-explored territory. Apache Kafka leads the way, which is why ksqlDB is built directly on top of it.
+An event is anything that occurred and was recorded. It could be something
+high-level that happened in a business, like the sale of an item or the
+submission of an invoice. Or it could be something low-level, like a log line
+emitted by a web server when a request is received. Anything that happens at
+a point in time is an event.
 
-Apache Kafka is a distributed streaming platform for working with events. It’s horizontally scalable, fault-tolerant, and wicked fast. Although working with it directly can be low level, it has an excellent opinionated approach for modeling both individual events and groups of them together. For this reason, ksqlDB heavily borrows some of Kafka’s abstractions. It doesn’t aim to make you learn all of Kafka, but it also doesn’t reinvent the wheel where there’s already something really good to use.
+Because events are so fundamental to stream processing, they are ksqlDB's core
+unit of data. All of ksqlDB's features are oriented around making it easy to
+solve problems using events. Although it's easy to think about individual
+events, figuring out how to group related events together is a bit more
```

Comment:
relating back to the comment on the other PR .. I was also suggesting grouped -> stored, to make it clearer when we discuss aggregating/grouping here ultimately

Reply:
Cool. +1. lll use that language going forward.